src/vscode-bicep/package.json

```diff
@@ -106,7 +106,17 @@
           "bicep"
         ],
         "extensions": [
-          ".bicep",
+          ".bicep"
+        ],
+        "configuration": "./language-configuration.json"
+      },
+      {
```

Comment:
You'll probably need to add to the `grammars` section of this file if you want to support a textmate fallback language

Reply:
Yeah, we should create that grammar. I opened #8072 to track that.